examples/rosa-classic-public-with-shared-vpc/variables.tf

```diff
@@ -1,57 +1,38 @@
 variable "cluster_name" {
-  type = string
-}
-
-variable "shared_vpc_aws_access_key_id" {
-  type = string
-}
-
-variable "shared_vpc_aws_secret_access_key" {
-  type = string
-}
-
-variable "shared_vpc_aws_region" {
-  type = string
-}
-
-variable "shared_vpc_role_name" {
-  type    = string
-  default = null
+  type        = string
+  description = "Name of the cluster. Cannot exceed 15 characters in length. After the creation of the resource, it is not possible to update the attribute value."
 }
 
 variable "openshift_version" {
-  type    = string
-  default = "4.14.9"
+  type        = string
+  default     = "4.14.9"
+  description = "Desired version of OpenShift for the cluster, for example '4.1.0'. If version is greater than the currently running version, an upgrade will be scheduled."
   validation {
     condition     = can(regex("^[0-9]*[0-9]+.[0-9]*[0-9]+.[0-9]*[0-9]+$", var.openshift_version))
     error_message = "openshift_version must be with structure <major>.<minor>.<patch> (for example 4.13.6)."
   }
 }
 
-variable "account_role_prefix" {
-  type    = string
-  default = null
-}
-
-variable "operator_role_prefix" {
-  type    = string
-  default = null
+variable "shared_vpc_aws_access_key_id" {
```

Comment:
Not too much familiar with shared vpc, why is this needed?

Reply:
The shared VPC is created on different AWS account.
This is the way to path the credentials of this account, in order to create all relevant resources in it.
The AWS account of the cluster is the passed as the default AWS account

Reply:
Oh this is to create the shared vpc